zerver/lib/actions.py

```diff
@@ -231,24 +231,26 @@ def send_signup_message(sender, signups_stream, user_profile,
         internal_blurb = " "
 
     user_count = realm_user_count(user_profile.realm)
-    notifications_stream = user_profile.realm.get_notifications_stream()
+    signups_stream = user_profile.realm.get_signups_stream()
     # Send notification to realm notifications stream if it exists
     # Don't send notification for the first user in a realm
-    if notifications_stream is not None and user_count > 1:
+    if signups_stream is not None and user_count > 1:
         internal_send_message(
             user_profile.realm,
             sender,
             "stream",
-            notifications_stream.name,
-            "New users", "%s just signed up for Zulip. Say hello!" % (
-                user_profile.full_name,)
+            signups_stream.name,
+            "New users",
+            "%s just signed up for Zulip!" % (
```

Comment:
I think the format of the admin_realm message is nice 
"%s <`%s`> just signed up for Zulip! (total: **%i**)"